samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -96,11 +96,25 @@ def _load_pipeline_bootstrap_context() -> Dict:
 
     config = SamConfig(PIPELINE_CONFIG_DIR, PIPELINE_CONFIG_FILENAME)
     if not config.exists():
+        context[str(["environment_names_message"])] = ""
         return context
-    for env in config.get_env_names():
+
+    # config.get_env_names() will return the list of
+    # bootstrapped env names and "default" which is used to store shared values
```

Comment:
I thought it returns a `version` key as well.

Reply:
nope it doesn't

Reply:
It only returns keys with its value being a dictionary

Reply:
Check: samcli/lib/config/samconfig.py:44

Reply:
https://github.com/aws/aws-sam-cli/blob/57f1b441f1c9730a3b81d044a68930bda9bb6800/samcli/lib/config/samconfig.py#L44-L48